src/modules/users/users.service.ts

```diff
@@ -0,0 +1,52 @@
+import { User, UserWithoutId } from './interfaces/user';
+import { userRepository } from '../users/users.repository';
+import { ERROR_MESSAGES } from '../../errors';
+import { UserParameters } from './interfaces/parameters';
+
+class UserService {
+  async create(user: UserWithoutId): Promise<User | Error> {
```

Comment:
your code doesn't return an Error in this method.
return !== throw

- [ ] remove Error as a return type